ui/components/multichain/token-list-item/token-list-item.js

```diff
@@ -82,7 +85,39 @@ export const TokenListItem = ({
     title === CURRENCY_SYMBOLS.ETH && isOriginalTokenSymbol
       ? t('networkNameEthereum')
       : title;
-
+  const stakeableTitle = (
+    <Box
+      data-testid={`staking-entrypoint-${chainId}`}
+      display={Display.InlineFlex}
+      flexDirection={FlexDirection.Row}
+      alignItems={AlignItems.center}
+      gap={1}
+      onClick={(e) => {
```

Comment:
UI wise looks good one question before I approve. On click of this stake button, it will only add it to metametrics? Shouldn't this button navigate the user to Staking?